docs/topics/braintree.rst

```diff
@@ -261,3 +260,27 @@ Some information about the subscripton is stored in solitude.
     :query paymethod__braintree_buyer: the primary key of the braintree buyer.
     :query paymethod__braintree_buyer__buyer: the primary key of the buyer.
     :query seller_product: the primary key of the product.
+
+Webhook
+-------
+
+When Braintree completes certain actions, webhooks will be sent. For more information
+see the `Braintree documentation <https://developers.braintreepayments.com/javascript+python/reference/general/webhooks>`_.
+
+.. http:get:: /braintree/webhook/verify/
+
+    .. code-block:: json
+
+        "some-token"
+
+    :<json bt_challenge: the bt_challenge issued by Braintree.
```

Comment:
It is a GET, but bear in mind that the request comes to payments service which then sends the request on solitude.

Reading the payments service PR, it is indeed sending a query param, will change.

https://github.com/mozilla/payments-service/pull/21/files#diff-340a6a494441afe57ca1271942b853c3R112